src/kv/deserialise.h

```diff
@@ -118,7 +118,9 @@ namespace kv
 
       if (history)
       {
-        history->append(data);
+        history->append(
+          data.data() + kv::serialised_entry_header_size,
+          data.size() - kv::serialised_entry_header_size);
```

Comment:
@jumaffre This is causing the history_bench to crash because it's inserting entries that are smaller than serialised_entry_header_size. The easiest fix is probably just to make them larger.

Reply:
Good spot! I didn't realise that these benches are no longer built/run by default (guarded by our cmake's `LONG_TESTS` flag).